src/bodywork/workflow_execution.py

```diff
@@ -51,23 +52,21 @@
 
 
 def run_workflow(
-    namespace: str,
```

Comment:
Should we not keep this here and set it to `BODYWORK_DEPLOYMENT_JOBS_NAMESPACE` at the CLI level.